apps/judicial-system/web/src/routes/Court/Indictments/Completed/Completed.tsx

```diff
@@ -508,21 +524,6 @@ const Completed: FC = () => {
             })}
           </Box>
         )}
-        {features?.includes(Feature.SERVICE_PORTAL) &&
-          workingCase.indictmentRulingDecision ===
-            CaseIndictmentRulingDecision.RULING && (
-            <Box>
-              <SectionHeading title={'Dómsorð'} marginBottom={2} heading="h4" />
-              <RulingInput
```

Comment:
Why are we removing this input?

Reply:
The ruling is now entered on the newly created court record screen. This input was hidden behind a feature flag so users won't notice that we removed it from here. See design 👇

https://www.figma.com/design/RVuZk7wG4xS53iwNM5cU78/R%C3%A9ttarv%C3%B6rslug%C3%A1tt---DMR?node-id=55253-50175&m=dev&t=bPQAA1hJpSqEl4Gl-1